Doc/tutorial/introduction.rst

```diff
@@ -182,6 +182,29 @@ the first quote::
    >>> print(r'C:\some\name')  # note the r before the quote
    C:\some\name
 
+There is one subtle aspect to raw strings that is of special concern to Windows
+programmers:  a raw string may not end in an odd number of ``\`` characters.
+This is because the interpreter sees this as an escaped quote character, and so
+it does not recognize it as the end of the string::
+
+   >>> fn = r'C:\this\will\not\work\'
+     File "<stdin>", line 1
+       fn = r'C:\this\will\not\work\'
+                                    ^
+   SyntaxError: EOL while scanning string literal
+
+There are several workarounds for this.  One is to use regular strings and
+double the backslashes::
+
+   >>> fn = 'C:\\this\\will\\work\\'
+
+Another is to add a blank character before the quote and then remove it::
+
+   >>> fn = r'C:\this\will\work\ '.strip()
```

Comment:
Personally I wouldn't mention this one -- it adds a runtime cost if it's inside some loop or function call, which on its own isn't a huge deal, but it also means you get unexpected behavior if you don't fully realize what this does and try it on `r'     foo bar \ baz quux\ `.

I'd personally leave things with the last workaround (and not mention there are multiple), but if another workaround is desired, using two string literals with the terminal slash in a second one implicitly concatenated is the next-best-one to me:

```
    fn = fn = r'C:\this\will\work' '\\'
```